resources/js/pages/demos/dialog/with-confirmation.tsx

```
@@ -0,0 +1,89 @@
+import { Head } from '@inertiajs/react'
+import { DemoContainer } from '@/components/docs/DemoContainer'
+import { Button } from '@/components/twc-ui/button'
+import { FormComboBox } from '@/components/twc-ui/combo-box'
+import { Form, useForm } from '@/components/twc-ui/form'
+import { FormGrid } from '@/components/twc-ui/form-grid'
+import { FormNumberField } from '@/components/twc-ui/number-field'
```

Biome (2.1.2)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)